### ISSUE TITLE
Update vis tool documentation

### ISSUE THREAD

**Issue by s-sd** [DeepRegNet/DeepReg#564]
## Vis tool docs

What are you trying to do and which part of the code is hard to understand or incorrect
or missing? Please provide a link to the readthedocs page and code if available.

As mentioned in #474.

The documentation for the vis tool needs to be updated with some images that are produced using the tool and possibly changing from mode numbers (such as `0`, `1`, `2`, `3`) to informative names (such as `gif_slices`, `gif_slices_tiled`, `gif_warp`, `plot_slices`)

If the documentation request is approved, would you be willing to submit a PR? _(Help
can be provided if you need assistance submitting a PR)_

Yes

**Comment by YipengHu** [comment]
@mathpluscode @s-sd need a decision on this. Sorry for the effort put in @s-sd , should we mark this to be re-visited after r1?

**Comment by s-sd** [comment]
@YipengHu for this ticket, I was waiting on a decision on if we want to move the demos to a separate repo or not (as we were discussing in #629). I was planning to add GIFs to the docs for clarity but again there's the issue of repo size so I would say that we can wait on this until we finalise the decision/ details about moving demos to a new repo. So yes, good to re-visit after r1, I'd say.

**Comment by s-sd** [comment]
Please see discussion in Issue #629. The repo size is increased if gifs are added, therefore, this issue is being closed for now. A new issue can be opened to change the vis tool modes from numbers to names if needed.
